formats/json/commonMain/src/kotlinx/serialization/json/internal/StringOps.kt

```diff
@@ -31,7 +33,7 @@ internal val ESCAPE_STRINGS: Array<String?> = arrayOfNulls<String>(93).apply {
 }
 
 @SharedImmutable
-internal val ESCAPE_MARKERS: ByteArray = ByteArray(93).apply {
+public val ESCAPE_MARKERS: ByteArray = ByteArray(93).apply {
```

Comment:
I've looked into it and it seems that this val isn't used anywhere in public, so it can be reverted back to internal